docs/dns.md

```diff
@@ -1,12 +1,11 @@
 # DNS Support
-If you point a domain or subdomain to Google Cloud DNS, you can use `kctf-setup-dns` and `kctf-chal-dns` to map *[challenge_name].yourdomain.com* to your challenge.
+If you point a domain or subdomain to Google Cloud DNS as an NS record, kCTF creates a DNS zone that maps the challenge to your challenge's IP address.
 
 Follow these steps:
 
-1. Configure the subdomain in `kctf-setup-config-create`. 
+1. Configure the domain in `kctf-setup-config-create --domain-name foo.example.com`. 
 
-   Example: If you enter *foo.example.com*, the challenges will be deployed on *[challenge-name].foo.example.com*.
-1. Run `kctf-setup-dns` to create the DNS zone in Google Cloud DNS.
+   Example: If you enter *foo.example.com*, the challenges are be deployed on *xxxxx.foo.example.com*.
```

Comment:
unless if it's a web challenge, then it's at www.xxxxx.foo.example.com

Reply:
(this goes to alpha)